ui/packages/consul-ui/app/components/consul/upstream-instance/list/index.hbs

```diff
@@ -1,4 +1,4 @@
-<ul data-test-proxy-upstreams>
+<ul data-test-proxy-upstreams class="proxy-upstreams">
```

Comment:
Could we take the opportunity here to name this `class="consul-upstream-instance-list"` (the name of the component) and add a `...attributes`.

I think here we could also move the component itself to a glimmer component, I think we should be able to just remove the js file for the component.

What do you think about moving the CSS into this folder also? Ah actually I just noticed, does this list need a grey top border?

Reply:
- Update class name
- Made it into a glimmer component
- CSS didn't need to be moved because there really isn't any. This list is extending the styling of composite-rows
- No grey border is missing. The search and sort is missing.